credsweeper/scanner/scan_type/pem_key_pattern.py

```diff
@@ -58,15 +60,12 @@ def is_pem_key(cls, lines: List[str], config: Config) -> bool:
         lines = cls.strip_lines(lines)
         lines = cls.remove_leading_config_lines(lines)
         key_data = ""
-        for line_num, line in enumerate(lines):
-            if line_num >= 190:
-                return False
```

Comment:
``` python
if len(lines) >= 190:
    return False
```

Reply:
Opposite. It will skip huge files. 
```remove_leading_config_lines`` does not trim ending so there might be a long tail over 190 lines with remaining.
Additionally, enumerate does additional complexity. 
If the key will be not found in given 190 lines - it returns False at end of function.
Probably there should be a huge refactoring...